frontend/src/components/ImplementationReportView/BadgeFormats.tsx

```diff
@@ -0,0 +1,65 @@
+export interface BadgeFormatOption {
+  type: string;
+  renderDisplay: (badgeURI: string) => JSX.Element;
+  generateCopyText: (badgeURI: string) => string;
+}
+
+// Supported format list
+export const badgeFormatOptions: BadgeFormatOption[] = [
+  {
+    type: "URL",
+    renderDisplay: (badgeURI: string): JSX.Element => (
+      <span>
+        <pre className="pt-2 pb-2">{badgeURI}</pre>
+      </span>
+    ),
+    generateCopyText: (badgeURI: string): string => `${badgeURI}`,
+  },
+  {
+    type: "Markdown",
+    renderDisplay: (badgeURI: string): JSX.Element => (
+      <span>
+        <pre className="pt-2 pb-2">![Static Badge]({badgeURI})</pre>
```

Comment:
What do you mean here by `Static Badge`? The alt text of the badge should likely depend on which badge we're showing, and probably match the name of the badge.

Reply:
This can be fixed by passing `prettyName` into renderDisplay and `generateCopyText` functions.